lib/tariff_synchronizer/file_service.rb

```diff
@@ -67,6 +55,15 @@ def download_content(url)
 
       private
 
+      def store_file_in_production(path, body)
```

Comment:
I would call this `store_file`, and then branch the code with the rails env in this method, rather than using send and two methods.